publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+# Results of accessibility self-review for Web of Things (WoT) Architecture
+
+Answers to accessibility (FAST) review checklist [(from here)](https://w3c.github.io/apa/fast/checklist.html)
+for the Web of Things (WoT) architecture as a whole.
+These answers are generally written to be applicable to all WoT deliverables.
+Where a response is specifically relevant to a particular deliverable that will be identified.
+
+## If technology allows visual rendering of content
+The WoT architecture does not directly support visual rendering of content.
+Some devices adhering to WoT standards may have visual display aspects, for example
+light bulbs that can be turned on or off, or small displays.
+However, WoT in general has no provision to constrain visual displays, 
+as this (and other controls limiting the behavior of devices)
+was not in the scope of our current charter.
+
+### [no] There is a defined way for a non-visual rendering to be created.	
+No.
+### [no] Content can be resized.	
+No specified mechanism.
+### [possible] Luminosity and hue contrast can adapt to user requirements.	
+No specified mechanism, although WoT TDs for lights usually have such controls, and WoT supports them.
+### [possible] Text presentation attributes can be changed.
+No specified mechanism, although WoT TDs may support controls.
+### [possible] Visual presentation of pointers and cursors can be adjusted.
+No specified mechanism, although WoT TDs may support controls.
+### [no] Changing content presentation does not render it unreadable.
+No specified mechanism for changing content presentation, and so no controls preventing unreadability.
+### [no] Technology does not allow blinking or flashing of content, or provides a feature for users to quickly turn it off or permanently disable it.
+WoT does not constrain device behaviour and in particular does not prevent devices from flashing. 
+In the future, the way to handle this (which was not in the scope of our current charter) would be
+to define a standard model for lights (and other devices with visual output) that included this feature.
+### [n/a] It is possible to make navigation order correspond to the visual presentation.
+Not applicable.
+
+## If technology provides author control over color
+Generally, WoT does not mandate mechanisms for control of color, although some
+devices adhering to WoT may be able to display color.
+As stated above, WoT in general has no provision to constrain visual displays, 
+as this (and other controls limiting the behavior of devices)
+was not in the scope of our current charter.
+
+### [no] There is a mechanism for users to override colors of text and user interface components.	
+Not supported.
+### [no] There is a feature for authors to define semantically available "color classes" that users can easily map to custom colors, and give preference to this vs. coloring objects individually.
```

Comment:
suggest n/a

Reply:
Left as "no", changed response to "not directly supported"... I guess this could be "possible", but since they are not asking if it is possible but if there "is a mechanism" I thought "no" made more sense.